rust/README.md

```diff
@@ -0,0 +1,110 @@
+# lib/rust
+
+Earthly's official collection of rust [UDCs](https://docs.earthly.dev/docs/guides/udc).
+
+## +CARGO
+
+This UDC runs the cargo command `cargo $args` caching the contents of `$CARGO_HOME/registry`, `$CARGO_HOME/git` and `target` for future builds of the same calling target. 
+
+### Usage
+
+First, import the UDC up in your Earthfile:
+```earthfile
+IMPORT github.com/earthly/lib/rust:<version/commit> AS rust
```

Comment:
Nit: `AS rust` isn't actually necessary when the name is the same. But also it doesn't hurt.

Reply:
I think it is more readable